src/viem/prepareTransactions.ts

```diff
@@ -372,7 +372,47 @@ export async function prepareTransferWithCommentTransaction(
   })
 }
 
-// TODO(ACT-956) create helper for native transfers
+/**
+ * Prepare a transaction for sending native asset.
+ *
+ * @param fromWalletAddress - sender address
+ * @param toWalletAddress - recipient address
+ * @param amount the amount of the token to send, denominated in the smallest units for that token
+ * @param feeCurrencies - tokens to consider using for paying the transaction fee
+ * @param sendToken - native asset to send. MUST be native asset (e.g. sendable using the 'value' field of a transaction, like ETH or CELO)
+ *
+ * @param prepareTxs a function that prepares the transactions (for unit testing-- should use default everywhere else)
+ **/
+export function prepareSendNativeAssetTransaction(
+  {
+    fromWalletAddress,
+    toWalletAddress,
+    amount,
+    feeCurrencies,
+    sendToken,
+  }: {
+    fromWalletAddress: string
+    toWalletAddress: string
+    amount: bigint
+    feeCurrencies: TokenBalance[]
+    sendToken: TokenBalance
```

Comment:
```suggestion
    sendToken: TokenBalance & {isNative: true}
```
should we enforce type here? Although I guess this opens up for handling a case in usePrepareSendTransactions for tokens that don't have an address and is also non native, which I believe should never happen?

Reply:
added static typing and an error in the logs if we get a token that isn't native and doesn't have an address

Reply:
> I guess this opens up for handling a case in usePrepareSendTransactions for tokens that don't have an address and is also non native, which I believe should never happen

we have tests in address-metadata to prevent this, so it should be ok